packages/react-sdk/src/components/Whiteboard/ElementBehaviors/ContextMenu/ElementContextMenu.tsx

```diff
@@ -128,16 +144,6 @@ function ContextMenuOptions({
 
   return (
     <Menu
-      componentsProps={{
```

Comment:
As far as I can read from the docs this is deprecated and should be replaced by `onClose`:
https://mui.com/material-ui/api/modal/